policy-controller/src/main.rs

```diff
@@ -72,13 +77,29 @@ async fn main() -> Result<()> {
             DETECT_TIMEOUT,
         );
 
-        tokio::spawn(index.run(client, ready_tx));
+        tokio::spawn(index.run(client.clone(), ready_tx));
         handle
     };
 
     // Run the gRPC server, serving results by looking up against the index handle.
     tokio::spawn(grpc(grpc_addr, cluster_networks, handle, drain_rx));
 
+    // Run the admission controller
+    let admission = linkerd_policy_controller::admission::Admission(client);
+
+    let routes = warp::path::end()
+        .and(warp::body::json())
+        .and(warp::any().map(move || admission.clone()))
+        .and_then(linkerd_policy_controller::admission::mutate_handler)
+        .with(warp::trace::request());
+    tokio::spawn(
+        warp::serve(warp::post().and(routes))
+            .tls()
+            .cert_path("/var/run/linkerd/tls/tls.crt")
+            .key_path("/var/run/linkerd/tls/tls.key")
```

Comment:
nit: probably best to have this directory configurable via args (as a PathBuf`)... not that it's really easy to run this in a nonstandard place but just to avoid harcoding. not a blocker.